### PR TITLE
odim-6871: Resolve the vulnerabilities found in github code scanning

### PR DESCRIPTION
**Pull request by hemkapoor**
Resolve the vulnerabilities found in github code scanning

### CODE REVIEW

**Review by AngithCJ — APPROVED** [review]

**Review by chauberahul1993 — APPROVED** [review]

File: odim-controller/scripts/odim-controller.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def perform_checks(skip_opt_param_check=False):
 		else:
 			REDIS_INMEMORY_PW_FILE = CONTROLLER_CONF_DATA['redisInMemoryPasswordFilePath']
 			if not os.path.exists(REDIS_INMEMORY_PW_FILE):
-				logger.critical("%s does not exist, exiting!!!", REDIS_INMEMORY_PW_FILE)
+				logger.critical("Redis In-Memory Password File Path does not exist, exiting!!!")
```
Comment (amar-shalgar) [inline odim-controller/scripts/odim-controller.py]: Please look into alphabet casing

File: odim-controller/scripts/odim-controller.py [hunk 2]
```diff
@@ -266,7 +266,7 @@ def perform_checks(skip_opt_param_check=False):
 		else:
 			REDIS_ONDISK_PW_FILE = CONTROLLER_CONF_DATA['redisOnDiskPasswordFilePath']
 			if not os.path.exists(REDIS_ONDISK_PW_FILE):
-				logger.critical("%s does not exist, exiting!!!", REDIS_ONDISK_PW_FILE)
+				logger.critical("Redis On-Disk Password File Path does not exist, exiting!!!")
```
Comment (amar-shalgar) [inline odim-controller/scripts/odim-controller.py]: Look into alphabet casing

File: lib-utilities/logs/logging.go [hunk 3]
```diff
@@ -116,10 +116,10 @@ func (l *Logging) auditLogEntry(ctx iris.Context, reqBody map[string]interface{}
 	reqStr := MaskRequestBody(reqBody)
 
 	// formatting logs in syslog format
-	if reqStr == "null" {
+	if reqStr == "" {
```
Comment (amar-shalgar) [inline lib-utilities/logs/logging.go:119]: This change may require changes in https://github.com/ODIM-Project/ODIM/blob/development/lib-utilities/logs/logging.go#L142

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: lib-utilities/logs/logging.go (hunk 3 above)
Comment (hemkapoor) [reply lib-utilities/logs/logging.go:119]: Already removed

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**